pkg/util/tags.go

```diff
@@ -0,0 +1,73 @@
+package util
+
+import (
+	ackutil "github.com/aws-controllers-k8s/runtime/pkg/util"
+	svcsdk "github.com/aws/aws-sdk-go/service/memorydb"
+
+	svcapitypes "github.com/aws-controllers-k8s/memorydb-controller/apis/v1alpha1"
+)
+
+func ComputeTagsDelta(
+	desired []*svcapitypes.Tag,
+	latest []*svcapitypes.Tag,
+) (addedOrUpdated []*svcapitypes.Tag, removed []*string) {
+	var visitedIndexes []string
+	var hasSameKey bool
+
+	for _, latestElement := range latest {
+		hasSameKey = false
+		visitedIndexes = append(visitedIndexes, *latestElement.Key)
+		for _, desiredElement := range desired {
+			if equalStrings(latestElement.Key, desiredElement.Key) {
+				hasSameKey = true
+				if !equalStrings(latestElement.Value, desiredElement.Value) {
+					addedOrUpdated = append(addedOrUpdated, desiredElement)
+				}
+				break
+			}
+		}
+		if hasSameKey {
+			continue
+		}
+		removed = append(removed, latestElement.Key)
+	}
+	for _, desiredElement := range desired {
+		if !ackutil.InStrings(*desiredElement.Key, visitedIndexes) {
+			addedOrUpdated = append(addedOrUpdated, desiredElement)
+		}
+	}
+	return addedOrUpdated, removed
+}
+
+func SdkTagsFromResourceTags(
```

Comment:
Please update to `SDK` to match the other
```suggestion
func SDKTagsFromResourceTags(
```

Reply:
Updated!